templates/constraints/yml/actions/fileCreateOrUpdateJsonFile.mustache

```diff
@@ -5,6 +5,32 @@
       target: ./Properties/launchSettings.json
       content:
         profiles:
+        {{#profiles}}
+        {{#profileName}}
```

Comment:
Maybe we don't need this condition. Just use
{{#profiles}}
  {{{profileName}}}
  commandName: "Project"
   ...
{{/profiles}}
{{^profiles}}
...

Reply:
Thanks, I'll update in the next PR.